cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1737,7 +1737,7 @@
 
   <property>
     <name>security.store.provider</name>
-    <value>kms</value>
+    <value>file</value>
```

Comment:
Yes, this needs more work. Closing it for now.

On Thu, Aug 11, 2016 at 9:02 AM Chris Gianelloni <notifications@github.com>
wrote:

> In cdap-common/src/main/resources/cdap-default.xml
> <https://github.com/caskdata/cdap/pull/6387#discussion_r74451052>:
>
> > @@ -1737,7 +1737,7 @@
> >
> >    <property>
> >      <name>security.store.provider</name>
> > -    <value>kms</value>
> > +    <value>file</value>
>
> Since this requires configuration to work, shouldn't this be none?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/caskdata/cdap/pull/6387/files/2a33dccfe3fc23bb66aa483fdfda05f2f05d8a1e#r74451052>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFGk_N7-A-K2TDdzk9pb2TVX3o7VOi5ks5qe0dxgaJpZM4Jhu3->
> .
>